data/db-schema/src/main/resources/net/corda/db/schema/vnode-vault/migration/vnode-vault-creation-v1.0.xml

```diff
@@ -28,6 +28,13 @@
             <column name="usage"/>
         </createIndex>
 
+        <createTable tableName="vnode_allowed_certificate">
```

Comment:
Will this have any usage outside of mutual TLS? If not, it could be good to rename this to something like "vnode_mtls_allowed_certificate"